tests/v2/validation/schemas/schemas_test.go

```diff
@@ -0,0 +1,322 @@
+
+	log.Info("Verify that the version in each schema is the preferred version.")
+	failedSchemaPreferredVersionCheck, err := checkPreferredVersion(sc.client, schemasCollection, sc.cluster.ID)
+	require.NoError(sc.T(), err)
+	require.Emptyf(sc.T(), failedSchemaPreferredVersionCheck, "Failed preferred version checks: %v", failedSchemaPreferredVersionCheck)
+}
+
+func (sc *SchemaChangesTestSuite) TestCRDCreateDeleteOperation() {
+	subSession := sc.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a new Custom Resource Definition (CRD).")
+	err := createCRD(sc.client, crdCreateFilePath)
+	require.NoError(sc.T(), err)
+
+	log.Info("Access the schemas endpoint for the newly created CRD and verify that it can be accessed successfully without any errors.")
+	err = kwait.PollUntilContextTimeout(context.Background(), twoSecondTimeout, defaults.FiveSecondTimeout, false, func(ctx context.Context) (done bool, pollErr error) {
+		_, pollErr = getSchemaByID(sc.client, localCluster, customSchemaID)
+		if pollErr != nil {
+			return false, pollErr
+		}
+		return true, nil
+	})
+	schemaResponse, err := getSchemaByID(sc.client, localCluster, customSchemaID)
+	require.NoError(sc.T(), err)
+
+	log.Info("Verify that the schema's resourceFields field is set to null.")
+	id := schemaResponse["id"].(string)
+	require.Nil(sc.T(), schemaResponse["resourceFields"], "ResourceFields should be null for schema "+id)
+
+	log.Info("Verify that the schemas endpoint for the new CRD does not list the child schemas and trying to access any of the child schemas results in a '404 Not Found' error.")
+	childSchemas := getChildSchemasForCronTab()
+	err = checkChildSchemasNotListed(sc.client, localCluster, childSchemas, schemaResponse)
+	require.NoError(sc.T(), err)
+
+	log.Info("Verify that the data returned from the schemas endpoint has the resourceMethods and collectionMethods fields.")
+	require.NotNil(sc.T(), schemaResponse["resourceMethods"], "resourceMethods should be present in this schema")
+	require.NotNil(sc.T(), schemaResponse["collectionMethods"], "collectionMethods should be present in this schema")
+
+	log.Info("Access the schema definition for the new CRD and verify that it can be accessed successfully without any errors.")
+	err = kwait.PollUntilContextTimeout(context.Background(), twoSecondTimeout, defaults.FiveSecondTimeout, false, func(ctx context.Context) (done bool, pollErr error) {
+		_, pollErr = getSchemaDefinitionByID(sc.client, localCluster, customSchemaID)
+		if pollErr != nil {
+			return false, pollErr
+		}
+		return true, nil
+	})
+	schemaResponse, err = getSchemaDefinitionByID(sc.client, localCluster, customSchemaID)
+	require.NoError(sc.T(), err)
+
+	log.Info("Verify that the data returned has definitions with resourceFields for each definition type, and that the resourceMethods and collectionMethods fields are present. Also, ensure that fields of type 'integer' in the YAML are parsed as 'int' in the schema definitions.")
```

Comment:
Same here, can we condense the log `Verify that the data returned has definitions with resourceFields for each definition type, and that the resourceMethods and collectionMethods fields are present`
This is too big of a log for the tests. and since we are checking those as validations, we do not have to add `Also, ensure that fields of type 'integer' in the YAML are parsed as 'int' in the schema definitions` in the log IMO.